packages/ui/v2/core/layouts/SettingsLayout.tsx

```diff
@@ -311,6 +313,12 @@ export default function SettingsLayout({
     };
   }, []);
 
+  useEffect(() => {
+    if (sideContainerOpen) {
+      setSideContainerOpen(!sideContainerOpen);
+    }
+  }, [router.asPath]);
```

Comment:
missing dependency?

Reply:
I understand the linting suggestion but I only want this to run when the router path changes. When both dependencies are added, the menu always closes because it triggers when `sideContainerOpen` changes but if it's open it'll just close automatically.

https://www.loom.com/share/7d453d125c6e4d758e9dcf0447db4345